drivers/sensor/apds9960/apds9960.h

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SENSOR_APDS9960_H_
+#define _SENSOR_APDS9960_H_
+
+#include <device.h>
```

Comment:
I have included all required header file here because only one header need to be added in .c file. Is it fine ?